app/models/cloud_volume_backup.rb

```diff
@@ -1,9 +1,61 @@
 class CloudVolumeBackup < ApplicationRecord
   include NewWithTypeStiMixin
+  include ProviderObjectMixin
+  include SupportsFeatureMixin
+  include CloudTenancyMixin
 
   acts_as_miq_taggable
 
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ExtManagementSystem"
   belongs_to :availability_zone
   belongs_to :cloud_volume
+  has_one :cloud_tenant, :through => :cloud_volume
+
+  def backup_restore_queue(userid, volumeid)
```

Comment:
If this method is on `CloudVolumeBackup` then I think just `restore_queue` would be a better method, `backup_restore_queue` is redundant.